src/test/suite/didact.test.ts

```diff
@@ -88,6 +93,61 @@ suite('Didact test suite', () => {
 		}
 	});
 
+	test('Scaffold new project, test for open: true file', async function () {
+		try {
+			await commandHandler.processInputs(testScaffoldOpen).then( async () => {
+				// test to make sure the groovy file, set to open: true in json, actually opens
+				try {
+					await waitUntil(() => {
+						return findEditorForFile(`simple.groovy`);
+					}, EDITOR_OPENED_TIMEOUT, 1000);
+				} catch (error) {
+					assert.fail(`simple.groovy has not been opened in editor`);
+				}
+			});
+		} catch (error) {
+			assert.fail(error);
+		}
+	});
+
+	test('Scaffold new project, test for standard file (no open)', async function () {
+		try {
+			await commandHandler.processInputs(testScaffoldOpen).then( async () => {
+				// test to make sure the other file, set to open: false in json, does not open
+				try {
+					await waitUntil(() => {
+						return findEditorForFile(`aThirdFile.txt`);
+					}, EDITOR_OPENED_TIMEOUT, 1000).then(() => {
+						assert.fail(`aThirdFile.txt was found opened in editor when it should not have been`);	
+					});
+				} catch (error) {
+					assert.ok(error);
+				}
+			});
+		} catch (error) {
+			assert.fail(error);
+		}
+	});
+
+	test('Scaffold new project, test for open: false file', async function () {
+		try {
+			await commandHandler.processInputs(testScaffoldOpen).then( async () => {
+				// test to make sure the other file, set to open: false in json, does not open
+				try {
```

Comment:
> And I know there is some code duplication in the new tests but was unable to quickly find a way to reduce it. Open to suggestions.

provided [PR](https://github.com/bfitzpat/vscode-didact/pull/4) to factorize these 2 methods.

for the other one, given that it is quite the  opposite that is checked, it doesn't seem easy to factorize